tests/test-utils/org.wso2.identity.apps.test.container/src/main/java/org/wso2/identity/apps/test/container/CypressTestResults.java

```diff
@@ -0,0 +1,98 @@
+/*
+ *Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.identity.apps.test.container;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Cypress test results class
+ */
+public class CypressTestResults {
+
+    private int numberOfTests;
+
+    private int numberOfPassingTests;
+
+    private int numberOfFailingTests;
+
+    private List<CypressTestSuite> suites = new ArrayList<>();
+
+    int getNumberOfTests() {
+
+        return numberOfTests;
+    }
+
+    void setNumberOfTests(int numberOfTests) {
```

Comment:
Let's have public access modifier for all getters and setters

Reply:
Fixed with 5479673